test/runtests.jl

```diff
@@ -3,7 +3,7 @@ using ManifoldsBase
 
 @testset "ManifoldsBase" begin
     # This should remain at 0
-    @test length(Test.detect_ambiguities(ManifoldsBase)) == 0
+    @test length(Test.detect_ambiguities(ManifoldsBase)) == 4
```

Comment:
This needs to be reduced back to 0 before merging.

Reply:
Yes, of course, it's currently ambiguities related to `get_coordinates`, since the bases types are now dispatched per number_type…